universe/System.cpp

```diff
@@ -194,10 +193,17 @@ const std::string& System::ApparentName(int empire_id, const Universe& u,
         if (blank_unexplored_and_none)
             return EMPTY_STRING;
 
-        if (m_star == StarType::INVALID_STAR_TYPE)
-            return UserString("UNEXPLORED_REGION");
-        else
-            return UserString("UNEXPLORED_SYSTEM");
+        if (m_star == StarType::INVALID_STAR_TYPE) {
+            std::string s;
+            s.append(m_name).append(UserString("UNEXPLORED_REGION"));
```

Comment:
for just two parts, you can just do return `m_name + UserString(...)`

Reply:
aye